README.md

```diff
@@ -136,7 +136,7 @@ Assuming your running environment is MacOS:
 
 3. Generate the CAPC cluster spec yaml file
     ```
-    clusterctl generate cluster \
+    clusterctl generate cluster capc-cluster \
```

Comment:
Is the cluster name param strictly needed, given that it's defined in the CLUSTER_NAME env var that precedes it?  If it is, then lets remove it from the env vars.  (clusterctl should substitute it into the template either way AFAIK).